azlinux-jupyter-nextflow/Dockerfile

```diff
@@ -0,0 +1,60 @@
+############################ Jupyter-nextflow With AzLinux ###################
```

Comment:
i haven't reviewed anything yet, but i think we will want to overwrite the existing Nextflow dockerfile, not add a new one - we will have no use for the old one once we merge this one

Reply:
I was proposing an idea to have both of them in qa-brh to test and see if anything is missing. Since, there were compromises during the installation, which I've mentioned as `CTDS comments` in the Docker files, it'd be easier to gauge the impact if we have them together. Can we keep both of them for a while and then demote the Ubuntu based Nextflow notebook overtime?

Reply:
we can use branches or tags for that